content/pt-br/auto_scaling.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Autoscaling
+status: Completed
+category: Propriedade
+---
+
+O Autoscaling é a capacidade de um sistema de [escalar](/escalabilidade/) automaticamente, normalmente, em termos de recursos de computação. Com um sistema de autoscaling, os recursos são adicionados automaticamente quando necessário e podem ser dimensionados para atender às demandas esporadicas dos usuários. O processo de autoscaling varia e é configurável para escalar com base em diferentes métricas, como memória ou tempo de processamento. Os serviços de nuvem gerenciados geralmente estão associados à funcionalidade de autoscaling, pois há mais opções e implementações disponíveis do que a maioria das implantações no local.
+
+No passado, a infraestrutura e os aplicativos eram arquitetados para considerar o pico de uso do sistema. Essa arquitetura significava que mais recursos eram subutilizados e não eram flexiveis às mudanças na demanda consumida. A inflexibilidade significou custos mais altos para o negócio e perda de negócios devido a interrupções devido à demanda excessiva.
```

Comment:
```suggestion
No passado, infraestrutura e aplicações eram arquitetados para considerar o pico de uso do sistema. Essa arquitetura significava que mais recursos eram subutilizados e não eram flexiveis às mudanças na demanda de usuários. Essa inflexibilidade significou custos mais altos para o negócio e perda de negócios devido a interrupções causadas pela demanda excessiva.
```
This last sentence I think it can be improved, not sure exactly how..
This is the original sentence:
```
The inelasticity meant higher costs to the business and lost business from outages due to overdemand.
```
cc @brunoguidone @edsoncelio